packages/app-builder/src/routes/_builder+/scenarios+/$scenarioId+/i+/$iterationId+/_edit-view+/_layout.tsx

```diff
@@ -238,17 +238,17 @@ function DeploymentActions({
         <CommitScenarioDraft scenarioId={scenario.id} iteration={iteration} />
       );
     case 'version':
-      if (iteration.status === 'required') {
+      if (iteration.status === 'ready_to_activate') {
```

Comment:
Better to display `activate` only when the status is `ready_to_activate`.

The old code is totally equivalent _for now_ but the day we push a "pending" state it won't.

I prefer this control flow structure